### PR TITLE
release 3.0.0

### LINKED ISSUE #5: Update maintainers
For the next release, remember to update the maintainers list removing my ID `kinow` :+1: (or before that too, but needs to release a new version/build)

### PR DESCRIPTION
**Pull request by oliver-sanders**
Got bored waiting for the autotick bot.

Also closes #5 

Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by oliver-sanders** [comment]
@conda-forge-admin, please rerender

**Review by MetRonnie — APPROVED** [review]
LGTM, only difference is the min Python version here is 3.7 compared to 3.6 in setup.py?

File: recipe/meta.yaml [hunk 1]
```diff
@@ -45,6 +46,5 @@ about:
 
 extra:
   recipe-maintainers:
-    - kinow
```
Comment (kinow) [inline recipe/meta.yaml:48]: :wave:

**Review by kinow — APPROVED** [review]
:+1:

File: recipe/meta.yaml (hunk 1 above)
Comment (oliver-sanders) [reply recipe/meta.yaml:48]: :wave:

**Comment by oliver-sanders** [comment]
> LGTM, only difference is the min Python version here is 3.7 compared to 3.6 in setup.py?

Good spot, 3.6 is [EOL now](https://endoflife.date/python) so we should drop support in a future release - https://github.com/metomi/isodatetime/issues/208

Could drop to 3.6 for this release, however, I think it's easier just to stick with 3.7 for now.

**Pull request merged**